.changeset/slimy-waves-design.md

```diff
@@ -0,0 +1,8 @@
+---
+"@astrouxds/angular": minor
+"@astrouxds/react": minor
+"@astrouxds/astro-web-components": minor
+"astro-website": minor
+---
+
+Created a new rux-tag component.
```

Comment:
@markacianfrani do you want me to list API and stuff here as well?

Reply:
naw I think its good enough to just call it out. the docs will be more useful